MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -4808,6 +4808,9 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 	initQueues(pCreateInfo);
 	reservePrivateData(pCreateInfo);
 
+	// Initialize fences for execution barriers
+	for (auto &stage: _barrierFences) for (auto &fence: stage) fence = [_physicalDevice->getMTLDevice() newFence];
```

Comment:
Could you give the fences labels like `[fence setLabel:[NSString stringWithFormat:@"%s Fence %d", stageName(stage), idx]]`?  Would be very convenient for debugging.

Reply:
Sure, pushed it.